src/app/components/co2-bar-chart/co2-bar-chart.component.ts

```diff
@@ -18,7 +18,7 @@ export class Co2BarChartComponent implements OnInit {
   
   co2BarChartColors = [
     { 
-      backgroundColor: '#ff3311',
+      backgroundColor: '#ff4233d8',
```

Comment:
the color value is invalid (i think)

hex color values only have 6 digits

Reply:
i guess the last two digits are for the alpha channel for transparency

Reply:
dang true, that a new feature in the css4 standard https://drafts.csswg.org/css-color/#hex-notation

Reply:
so it is